test/y2storage/planned/lvm_lv_test.rb

```diff
@@ -75,12 +75,70 @@
     end
 
     context "when size is a percentage" do
+      let(:container) { volume_group }
+
       before do
         lvm_lv.percent_size = 50
+        allow(container).to receive(:is?).with(:lvm_lv).and_return(lvm_lv?)
       end
 
-      it "returns the size based on the volume group size" do
-        expect(lvm_lv.size_in(volume_group)).to eq(15.GiB)
+      context "and the logical volume is on top of a volume group" do
```

Comment:
Probably it would be more precise: `and the logical volume is a not thin volume`.